src/cpp/ext/filters/census/client_filter.cc

```diff
@@ -222,15 +238,26 @@ void OpenCensusCallTracer::OpenCensusCallAttemptTracer::RecordEnd(
   }
 }
 
+CensusContext OpenCensusCallTracer::OpenCensusCallAttemptTracer::
+    CreateCensusContextForCallAttempt(OpenCensusCallTracer* call_tracer) {
```

Comment:
Why make this a method on `OpenCensusCallAttemptTracer` when it's actually only accessing data on `OpenCensusCallTracer`?  Seems like it could just be a method on `OpenCensusCallTracer`, and the `OpenCensusCallAttemptTracer` could just change `CreateCensusContextForCallAttempt(parent_)` to `parent_->CreateCensusContextForCallAttempt()`.

Reply:
Done